### ISSUE TITLE
BEP-12 compliance?

### ISSUE THREAD

**Issue by martelli** [anacrolix/torrent#344]
Hey,

From the function `startMissingTrackerScrapers()`: https://github.com/anacrolix/torrent/blob/a298d295b26bcfd25160d38178ae10abd8e6ed1c/torrent.go#L1296-L1306
I understand that BEP 12 is not honored, correct?

**Comment by anacrolix** [comment]
That's correct.

**Comment by martelli** [comment]
I'm willing to implement it. Any objections?

Maybe add a config flag to not break current users who rely on the current behavior?

**Comment by anacrolix** [comment]
Yes, please, go for it. A flag would be great. The biggest difficulty I foresee is how to treat magnet link trackers, it's not clear how they fit in, and the reason the client currently just treats the announce list as a set of urls.

**Comment by anacrolix** [comment]
Just found this, which may be of interest: https://github.com/bittorrent/bittorrent.org/issues/82

**Comment by anacrolix** [comment]
@martelli where are you at on this?
